src/main/java/org/teamresistance/frc/command/SnorfleCommand.java

```diff
@@ -0,0 +1,28 @@
+package org.teamresistance.frc.command;
+
+import org.strongback.command.Command;
+import org.strongback.components.Motor;
+
+/**
+ * @author Tarik C. Brown
+ */
+public class SnorfleCommand extends Command {
+  public final double snorfspeed;
+  public Motor snorfleMotor;
+
+  public SnorfleCommand(Motor snorfleMotor, double snorfspeed) {
```

Comment:
Thoughts on naming the noun **snorfler** and the verb **snorfle**? The motor would then become `snorflerMotor` and the speed `snorfleSpeed`.